src/main/java/frc/robot/RobotContainer.java

```diff
@@ -5,28 +5,28 @@
 package frc.robot;
 
 import com.kauailabs.navx.frc.AHRS;
+
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.trajectory.TrapezoidProfile;
 import edu.wpi.first.wpilibj.Joystick;
+import frc.robot.commands.RampMove;
+import frc.robot.commands.StaticClimb;
 import frc.robot.commands.drive.WheelAlign;
+import frc.robot.subsystems.Climber;
+import frc.robot.subsystems.Ramp;
 import frc.robot.subsystems.swervev2.KinematicsConversionConfig;
 import frc.robot.subsystems.swervev2.SwerveDrivetrain;
 import frc.robot.subsystems.swervev2.SwerveIdConfig;
 import frc.robot.subsystems.swervev2.SwervePidConfig;
-import edu.wpi.first.wpilibj2.command.Command;
-import frc.robot.commands.RampMove;
-import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
-import edu.wpi.first.wpilibj2.command.button.Trigger;
-import frc.robot.subsystems.Ramp;
 import frc.robot.utils.smartshuffleboard.SmartShuffleboard;
-import edu.wpi.first.wpilibj.XboxController;
 
 
 public class RobotContainer {
   private final Joystick joyleft = new Joystick(Constants.LEFT_JOYSICK_ID);
   private final Joystick joyright = new Joystick(Constants.RIGHT_JOYSTICK_ID);
   private final SwerveDrivetrain drivetrain;
+  private Climber climber = null;
```

Comment:
there is no need to initialize climber to null. Consider removing the null and making Climber final.